packages/shared/src/modules/activities/libs/types/finish-course-dto.type.ts

```diff
@@ -0,0 +1,8 @@
+type FinishCourseDto = {
```

Comment:
```suggestion
type ActivityFinishCourseDto = {
```
do not forget about file name

add response or request